website/pages/de/versions.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary');
+
+const Container = CompLibrary.Container;
+
+const CWD = process.cwd();
+
+const versions = require(`${CWD}/versions.json`).filter(version => version != "12.0");
+
+function Versions(props) {
+  const {config: siteConfig} = props;
+  const latestVersion = versions[0];
+  const repoUrl = `https://github.com/${siteConfig.organizationName}/${siteConfig.projectName}`;
+  return (
+    <div className="docMainWrapper wrapper">
+      <Container className="mainContainer versionsContainer">
+        <div className="post">
+          <header className="postHeader">
+            <h1>{siteConfig.title}</h1>
+          </header>
+          <h3 id="latest">Versionen</h3>
+          <table className="versions">
+            <tbody>
+              <tr>
+                <th>{latestVersion}</th>
+                <td>
+                  {/* You are supposed to change this href where appropriate
+                        Example: href="<baseUrl>/docs(/:language)/:id" */}
+                  <a
+                    href={`${siteConfig.baseUrl}${siteConfig.docsUrl}/${
+                      props.language ? props.language + '/' : ''
+                    }introduction`}>
+                    Handbuch/Bedienungsanleitung
+                  </a>
+                </td>
+              </tr>
+              {versions.map(
+                version =>
+                  version !== latestVersion && (
+                    <tr key={version}>
+                      <th>{version}</th>
+                      <td>
+                        {/* You are supposed to change this href where appropriate
+                        Example: href="<baseUrl>/docs(/:language)/:version/:id" */}
+                        <a
+                          href={`${siteConfig.baseUrl}${siteConfig.docsUrl}/${
+                            props.language ? props.language + '/' : ''
+                          }${version}/introduction`}>
+                          Handbuch/Bedienungsanleitung
+                        </a>
+                      </td>
+                    </tr>
+                  ),
+              )}
+            </tbody>
+          </table>
+        </div>
+      </Container>
+    </div>
+  );
+}
+
+module.exports = Versions;
```

Comment:
Can't we just put a function in the standard versions file that works this out with some translations?

Reply:
Yeah I think this would probably be worth doing. Added #288 for it